checker/src/org/checkerframework/checker/genericeffects/messages.properties

```diff
@@ -1,3 +1,5 @@
 override.effect.invalid=%s in %s cannot override %s in %s; a method override may only be a Super Effect if it overrides a method with Super Effect.
 constructor.call.invalid=Calling a constructor with %s effect from a context limited to %s effects.
 call.invalid.super.effect=Calling a method with %s effect from a context limited to %s effects.
```

Comment:
I'm not sure why the ```super``` bit is in here.  I know you didn't pick this name, but we should change it to ```call.invalid.effect```